SignalServiceKit/src/Contacts/ContactsUpdater.m

```diff
@@ -40,6 +40,10 @@ - (instancetype)init
 
 - (nullable SignalRecipient *)synchronousLookup:(NSString *)identifier error:(NSError **)error
```

Comment:
I'd like to rework message sending significantly, and one goal would be to remove this method.